x-pack/plugins/observability_ai_assistant/public/components/chat/chat_prompt_editor.tsx

```diff
@@ -107,9 +115,16 @@ export function ChatPromptEditor({ disabled, loading, onSubmit }: ChatPromptEdit
   }, [handleSubmit]);
 
   useEffect(() => {
-    if (ref.current) {
-      ref.current.focus();
+    const textarea = textAreaRef.current;
+
+    if (textarea) {
+      textarea.focus();
+      textarea.addEventListener('input', () => handleResizeTextArea(textarea), false);
     }
+
+    return () => {
+      textarea?.removeEventListener('input', () => handleResizeTextArea(textarea), false);
```

Comment:
This doesn't work I think, because you're creating a new reference by using an arrow function literal.